packages/ecmascript/package.js

```diff
@@ -0,0 +1,21 @@
+Package.describe({
+  name: 'ecmascript',
+  version: '0.0.1',
+  // Brief, one-line summary of the package.
+  summary: '',
+  // URL to the Git repository containing the source code for this package.
+  git: '',
+  // By default, Meteor will default to using README.md for documentation.
+  // To avoid submitting documentation, set this field to null.
+  documentation: 'README.md'
+});
+
+Package.registerBuildPlugin({
+  name: 'compile-ecmascript',
+  use: ['compiler-plugin', 'babel-compiler@5.6.17'],
```

Comment:
Once @dgreensp reviews https://rbcommons.com/s/meteor/r/56/ this will change to `isobuild:compiler-plugin@1.0.0`.